woocommerce/woocommerce-ios
# [Store creation] Add new fun facts to loading screen

Closes: #9445


## Description
Add the following fun facts to the store creation loading screen,
- Turning on the lights
- Opening the doors

## Testing instructions
- Depending upon how soon the site gets ready (network connection quality and retry attempts) you might not see these two new fun facts. 
- CI passing should be enough.

If you want to see the store creation loading screen,
- Launch the app and tap "Get Started"
- Login and start the store creation flow by tapping the "Add a Store" button from "Menu -> Select Store" screen.
- Select "Create a Store" enter a store name
- Skip the profiler questions and select a country
- Tap "Try For Free" to start the store creation flow to see the loading screen


## Screenshots

<img src="https://user-images.githubusercontent.com/524475/232441316-1eb2632e-867f-4f4f-9aa3-9616fa5bc37e.png" width="320"/>

---
